packages/common/index.ts

```diff
@@ -275,15 +275,44 @@ export class Question {
   location?: string
 }
 
-// Question Types
-export enum QuestionType {
-  Concept = 'Concept',
-  Clarification = 'Clarification',
-  Testing = 'Testing',
-  Bug = 'Bug',
-  Setup = 'Setup',
-  Other = 'Other',
-}
+export const QuestionTypes: AddQuestionTypeParams[] = [
+  {
+    id: 1,
+    cid: 1,
```

Comment:
Is it default values for question types? What does cid represent?

Reply:
These are just used in tests, cid is course id.

Reply:
If it's only used for tests, do we need it for common?